core/jobs/batch_jobs/exp_title_validation_jobs.py

```diff
@@ -0,0 +1,47 @@
+
+# coding: utf-8
+#
+# Copyright 2021 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Validation Jobs for title exploration"""
+
+from core.domain import exp_fetchers
+from core.jobs import base_jobs
+from core.jobs.io import ndb_io
+from core.jobs.types import job_run_result
+from core.platform import models
+
+import apache_beam as beam
+
+(exp_models, ) = models.Registry.import_models([models.NAMES.exploration])
+
+
+class GetNumberOfExpExceedsMaxTitleLengthJob(base_jobs.JobBase):
+
+    def run(self) -> beam.PCollection[job_run_result.JobRunResult]:
+        return (
+            self.pipeline
+            | 'Get all ExplorationModels' >> ndb_io.GetModels(
+                exp_models.ExplorationModel.get_all())
+            | 'Length of exploration title' >> beam.Map(self.get_length_of_exploration_title)
+            | 'Exploration title having length more than 36' >> (
+                beam.Filter(lambda title: len(title) > 36))
+            | 'Sum Values' >> beam.CombineGlobally(sum)
+            | 'Map as stdout' >> beam.Map(job_run_result.JobRunResult.as_stdout)
+        )
+
+    def get_length_of_exploration_title(self, model: exp_models.ExplorationModel) -> int:
+        exploration = exp_fetchers.get_exploration_from_model(model)
+        return len(exploration.title)
```

Comment:
Ah I found the mistake, here you return `len` of title but on line 40 you also get the `len` of title.

Reply:
Just realized the mistake. I'll update it and check the output.
Thanks
Hitesh